### PR TITLE
Login maintenance and Billing scenarios

### PR DESCRIPTION
**Pull request by Atapiavel**

### CODE REVIEW

File: features/step_definition/login.spec.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const LoginPage = require('../../pages/login.pages.js');
 const ActionsPage = require('../../pages/actions.pages.js')
 const LoginPageLocator = require('../../locators/login.locators.js');
-const {Given, When, Then} = require('cucumber');
```
Comment (58mehrhas-wq) [inline features/step_definition/login.spec.js:4]: Are they not needed here? That's why you removed "Given" and "Then"?

File: features/support/asserts.js [hunk 2]
```diff
@@ -1,25 +0,0 @@
-const { Selector, test, t } = require('testcafe');
```
Comment (58mehrhas-wq) [inline features/support/asserts.js:1]: Any reason why you removed this file?

**Review by 58mehrhas-wq — COMMENTED** [review]
Everything looks good👍 Just put couple of comments and questions.

File: features/step_definition/login.spec.js (hunk 1 above)
Comment (Atapiavel) [reply features/step_definition/login.spec.js:4]: Yes, we don´t use both of them in this spec.

File: features/support/asserts.js (hunk 2 above)
Comment (Atapiavel) [reply features/support/asserts.js:1]: Asserts running from other files don´t break gherkin executions, we need to use assertions directly in the steps definition

**Pull request merged**